docs/source/oci/advanced/logging.rst

```diff
@@ -0,0 +1,49 @@
+.. _oci_logging:
+
+Logging and Output
+==================
+
+Writing Custodian Logs to Object Storage
+-----------------------------------------
+
+You can output logs and resource records to Object Storage.
+By default, Custodian will add the policy name and date as the prefix to the blob.:
+
+    .. code-block:: sh
+
+        custodian run -s oci://<Bucket_Name>/<Optional_Folder_Name>/ <policy file>
+
+In order to send data to a bucket in a different profile you can use query params
+
+    .. code-block:: sh
+
+        custodian run -s oci://<Bucket_Name>/<Optional_Folder_Name>/?profile=<NAME> <policy file>
+
+If the bucket does not exist then Custodian will try to create a bucket in the
```

Comment:
hmm.. I appreciate the out of the box here, but object storage doesn't quite feel like the right place versus an error if doesn't exist.  beyond typos,  there's too many constraints and considerations for larger orgs in my experience wrt for a simple bucket creation to be compliant.

Reply:
Understood. Changed to throw an error instead if the bucket is not found